proto-public/pbresource/resource.proto

```diff
@@ -181,6 +181,13 @@ message Reference {
   string section = 4;
 }
 
+// Given a resource, a tombstone represents a promise to delete all immediately
+// owned (child) resources, if any.
+message Tombstone {
+  // Owning resource identifier.
+  ID owner = 1;
+}
+
```

Comment:
nit (sorry!): Would you mind starting these comments with the type/field name, so that they're valid godoc comments?

```suggestion
// Tombstone represents a promise to delete all of a resource's immediately
// owned (child) resources, if any.
message Tombstone {
  // Owner identifies the deleted resource.
  ID owner = 1;
}

```

Reply:
TIL 😄